src/plugins/charts/inspector/SeriesOptions.vue

```diff
@@ -80,25 +85,46 @@ export default {
     mounted() {
         this.key = this.openmct.objects.makeKeyString(this.item);
         this.initColor();
-        this.unObserve = this.openmct.objects.observe(this.domainObject, `this.domainObject.configuration.barStyles[${this.key}]`, this.initColor);
+        this.unObserve = this.openmct.objects.observe(this.domainObject, `this.domainObject.configuration.barStyles.series[${this.key}]`, this.initColor);
     },
     beforeDestroy() {
         if (this.unObserve) {
             this.unObserve();
         }
     },
     methods: {
-        initColor() {
-            if (this.domainObject.configuration.barStyles && this.domainObject.configuration.barStyles[this.key]) {
-                this.currentColor = this.domainObject.configuration.barStyles[this.key].color;
-                this.name = this.domainObject.configuration.barStyles[this.key].name;
+        async initColor() {
+            // this is called before the plot is initialized
+            if (!this.domainObject.configuration.barStyles) {
+                this.domainObject.configuration.barStyles = {};
+                this.domainObject.configuration.barStyles.series = {};
+            }
+
+            const telemetryObject = await this.openmct.objects.get(this.key);
```

Comment:
Wondering if there is a way around having to call objects.get(...) here...probably not? Let me think.

Reply:
talked about storing it in the domainObject.configuration.barStyles instead when doing an addTelemetry in the plot, then reading it here in the inspector.